packages/react-composites/src/composites/CallComposite/components/CallPane.tsx

```diff
@@ -0,0 +1,195 @@
+    /* @conditional-compile-remove(call-composite-participant-pane) */
+    return localeStrings.strings.call;
+
+    return localeStrings.strings.callWithChat;
+  };
+
+  const strings = useStrings();
+  const theme = useTheme();
+
+  const header =
+    props.activePane === 'none' ? null : props.mobileView ? (
+      <TabHeader {...props} strings={strings} activeTab={props.activePane} />
+    ) : (
+      <SidePaneHeader
+        {...props}
+        strings={strings}
+        headingText={props.activePane === 'people' ? strings.peoplePaneTitle : ''}
+      />
+    );
+
+  /**
+   * In a Call Composite when a participant is removed, we must remove them from the call.
+   */
+  const removeParticipantFromCall = async (participantId: string): Promise<void> => {
+    await props.callAdapter.removeParticipant(participantId);
+  };
+
+  const peopleContent = (
+    <CallAdapterProvider adapter={props.callAdapter}>
+      <PeoplePaneContent
+        {...props}
+        onRemoveParticipant={removeParticipantFromCall}
+        setDrawerMenuItems={setDrawerMenuItems}
+        strings={strings}
+      />
+    </CallAdapterProvider>
+  );
+
+  // Use document.getElementById until Fluent's Stack supports componentRef property: https://github.com/microsoft/fluentui/issues/20410
+  const modalLayerHostElement = document.getElementById(props.modalLayerHostId);
+  const modalHostRef = useRef<HTMLElement>(modalLayerHostElement);
+  const modalHostWidth = _useContainerWidth(modalHostRef);
+  const modalHostHeight = _useContainerHeight(modalHostRef);
+  const minDragPosition: _ICoordinates | undefined = useMemo(
+    () =>
+      modalHostWidth === undefined
+        ? undefined
+        : {
+            x: props.rtl ? -1 * modalPipRightPositionPx : modalPipRightPositionPx - modalHostWidth + modalPipWidthPx,
+            y: -1 * modalPipTopPositionPx
+          },
+    [modalHostWidth, props.rtl]
+  );
+  const maxDragPosition: _ICoordinates | undefined = useMemo(
+    () =>
+      modalHostWidth === undefined || modalHostHeight === undefined
+        ? undefined
+        : {
+            x: props.rtl ? modalHostWidth - modalPipRightPositionPx - modalPipWidthPx : modalPipRightPositionPx,
+            y: modalHostHeight - modalPipTopPositionPx - modalPipHeightPx
+          },
+    [modalHostHeight, modalHostWidth, props.rtl]
+  );
+
+  const pipStyles = useMemo(() => getPipStyles(theme), [theme]);
+
+  const dataUiId = props.activePane === 'people' ? 'call-composite-people-pane' : '';
+
+  return (
+    <Stack verticalFill grow styles={paneStyles} data-ui-id={dataUiId} tokens={props.mobileView ? {} : sidePaneTokens}>
+      {header}
+      <Stack.Item verticalFill grow styles={paneBodyContainer}>
+        <Stack horizontal styles={scrollableContainer}>
+          <Stack.Item verticalFill styles={scrollableContainerContents}>
+            <Stack styles={props.activePane === 'people' ? availableSpaceStyles : hiddenStyles}>{peopleContent}</Stack>
```

Comment:
assigning a JSX element to `peopleContent` doesn't provide much value here since it is the only jsx being rendered here.
I would recommend moving the JSX inline.

Reply:
Great point! Moved it to a JSX inline.